packages/playwright/src/reporters/base.ts

```diff
@@ -385,10 +405,10 @@ export function formatFailure(screen: Screen, config: FullConfig, test: TestCase
           if (text.length > 300)
             text = text.slice(0, 300) + '...';
           for (const line of text.split('\n'))
-            resultLines.push(screen.colors.cyan(`    ${line}`));
+            resultLines.push(screen.colors.dim(`    ${line}`));
```

Comment:
Perhaps attachment content should not be dim, if we assume it is being read?

Reply:
I'd say dim as the log. If anything, dim is more readable than cyan.